src/common/adapters/jira.js

```diff
@@ -19,7 +19,7 @@ function isJiraPage(loc, doc) {
   return false;
 }
 
-const pathSuffixes = new RegExp('/(browse/[^/]+|projects/[^/]+/issues/[^/]+|secure/RapidBoard.jspa)$', 'g');
+const pathSuffixes = new RegExp('/(browse/[^/]+|projects/[^/]+/issues/[^/]+|secure/RapidBoard.jspa)|(jira/software/projects/[^/]+/boards/[^/]+/)$', 'g');
```

Comment:
Is the `jira/software/…` part new or optional? All the other example paths above don't seem to have that (even though the regexes would still match. So I'm wondering whether the regex should require the prefix. What happens if you remove `jira/software` from the URL in your browser? Does it still work?

Reply:
`jira/software/projects` is the path prefix when you browsing from the panel navigation jumping from Boards/Roadmap/Backlog views

Reply:
I see. Thanks @qclin. 💚 

The `RegExp` can be simplified slightly, to be:

```suggestion
const pathSuffixes = new RegExp('/(browse/[^/]+|projects/[^/]+/issues/[^/]+|secure/RapidBoard.jspa|jira/software/projects/[^/]+/boards/[^/]+/)$', 'g');
```

and we need a test for this.